packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.routing.js

```diff
@@ -0,0 +1,58 @@
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+
+import controller from './repayments.controller';
+import template from './repayments.html';
+
+const { stateParams } = ListLayoutHelper;
+const { sortOrder } = stateParams;
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('telecom.telephony.repayments', {
+      url: '/repayments',
+      views: {
+        'telephonyView@telecom.telephony': {
+          controller,
+          template,
+          controllerAs: '$ctrl',
+        },
+      },
+      resolve: {
+        breadcrumb: /* @ngInject */ ($translate) =>
+          $translate.instant('telephony_repayments_title'),
+        svaWalletLink: /* @ngInject */ (coreURLBuilder) =>
+          coreURLBuilder.buildURL('telecom', '#/telephony/sva-wallet'),
+        hasSvaWallet: /* @ngInject */ ($q, $state, svaWallet) => {
+          if (svaWallet && svaWallet.status === 'VALID') return true;
+          return $q.reject().catch(() => {
```

Comment:
looks the same as just returning :
```js
$state.go('telecom.telephony');
```

Reply:
:+1:  I though state redirections were prohibited inside router resolutions